.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,9 +14,9 @@ assignees: ''
 
 
 **Tell us about your environment:**  
-- pgMonitor Version:
-- PostgreSQL Version:  
 - Operating System:  
-- Exporter in use:  
-- Virtualization:  
-- Any customization:  
+- PostgreSQL Version:  
+- Ansible Version:  
+- Roles version or branch:  
+- Inventory and Playbook (names):  
+- Changes to stock Roles, Inventory, Playbook:
```

Comment:
Can we leave the pgMonitor version request here?

The stuff asking about Ansible roles/inventory/playbooks doesn't make much sense as a default request for this project.

Reply:
Agree with Keith, original list is good
Things to add 
- is this Container vs non container
- container based on which image (based os)
- Versions for Prometheus, Alertmanager, Grafana

Reply:
how's this new list?

Reply:
thoughts on the current iteration @sharmay ?

Reply:
updated based on your slack comments